openshift/kustomize/services/reporting/base/config-map.yaml

```diff
@@ -21,3 +21,4 @@ data:
   CHES_EMAIL_AUTHORIZED: "true"
   CHARTS_URL: http://charts-api:8080
   RESEND_ON_FAILURE: "true"
+  SEND_TO_ALL_BEFORE_FAILING: "false"
```

Comment:
We don't want it to try sending to everyone before failing.  It locks the service up for a long time.